docs/docs/deep-dive/mac.md

```diff
@@ -0,0 +1,19 @@
+# Tracee on Apple Mac FAQ
+
+## Does Tracee run on MacOS?
+
+No. Tracee runs only on Linux operating system. But you can run a Linux VM on your Mac, and then run Tracee in it. Please note though, that with this setup Tracee will only have visibility to the Linux VM it is running in, and not the host Mac machine.
+
+## Does Tracee run on Apple Silicone?
+
+Apple Silicone (a.k.a `M#` CPUs) are based on the ARM64 CPU architecture. Tracee does support ARM64 and therefore should run on Apple Silicone.
```

Comment:
Yep, I like the suggested change better. ARM64 == AARCH64, and Mx processors are AARCH64 as well (implementing ARMv8.5 architecture). So the problems isn't related to the architecture, but the Linuxkit.

Reply:
there's a separate section on Linuxkit compatibility. Why do we need to mention it under the cpu architecture compatibility?

Reply:
We don't, as long as the statement about the architecture is correct. Idea should be that a linux running on a virtual machine in MacOS would be ok as long as the OS is supported (provides BTF enabled kernel, with kallsyms available, etc).

Reply:
What I had in mind about the "Apple Silicon" topic is: Our arm64 binary runs on apple silicon because it is **just another aarch64 CPU**.

The "usual problem" on macs is related to MacOS and the virtual machine used (that might not have the requirements needed by tracee), and not the architecture.

If you don't think that should be clarified there, its ok as well.

Reply:
> The "usual problem" on macs is related to MacOS and the virtual machine used ... , and not the architecture.
If you don't think that should be clarified there, its ok as well

That's exactly what I'm saying. if the issue is unrelated to CPU architecture, why discuss it under this topic and not under the relevant topic

Reply:
ok based on this discussion, I'll accept the reworking suggestion but not the addendum on linuxkit. Thanks for reviewing!